simulation_node:
  ros__parameters:
    init_x: 0.0
    init_y: 0.0
    init_z: 0.0
    # 模型参数
    mesh_resource: "package://simulation_env/meshes/robot.dae"
    mesh_scale: 0.0005
    mesh_offset_x: -0.15  # 模型中心偏移
    mesh_offset_y: -0.16

    # 控制参数
    max_linear_vel: 1.0   # 最大线速度
    max_angular_vel: 3.14159  # 最大角速度 (π)
    control_rate: 50.0    # 控制频率 (Hz)

    # 点云参数
    pcd_file_path: "/home/tsm/simulation_ws/src/simulation_env/PCD/forest.pcd"
    search_radius: 0.2    # 邻近点搜索半径
    min_neighbors: 5     # 最小邻近点数

    # 高度估计参数
    height_offset: 0.0    # 高度偏移